Updates/841_Saltgurka-Honor_Hold_Updates.sql

```diff
@@ -0,0 +1,1092 @@
+UPDATE `creature_template` SET `EquipmentTemplateId`=1195 WHERE `entry`=16842;
+
+-- Also the human model was missing from their creature_template
+UPDATE `creature_template` SET `ModelId1`=16387,`ModelId2`=16388,`ModelId3`=16389,`ModelId4`=16390 WHERE `entry` IN (16842,20513); -- 16390
+
+-- Removing the gender link, because some creatures need to use the male model only
+UPDATE `creature_model_info` SET `modelid_other_gender`=0 WHERE `modelid` BETWEEN 16387 AND 16390;
+
+-- The guards outside the destroyed keep should always be male, as indicated by their texts. "I am a grumpy old man!"
+UPDATE `creature` SET `modelid`=16387 WHERE `guid` IN(72636,72637);
+
+-- Nethergarde Infantry were missing the female model
+UPDATE `creature_template` SET `ModelId1`=16377,`ModelId2`=16376,`ModelId3`=16375 WHERE `entry`=16831; 
+UPDATE `creature_model_info` SET `modelid_other_gender`=0 WHERE `modelid` IN(16375,16376,16377);
+UPDATE `creature_model_info` SET `modelid_alternative`=16377 WHERE `modelid`=16376; 
+
+-- Nethergarde infantry inside inn should have mugs and mutton in their hands.
+DELETE FROM `creature_equip_template` WHERE `entry` = 5593;
+INSERT INTO `creature_equip_template` (`entry`, `equipentry1`, `equipentry2`, `equipentry3`) VALUES
+(5593,2704,13407,0);
+
+-- The infantry that's doing work around the town and the guard at the outhouse should not be carrying any weapons. Override with an empty equip_template
+DELETE FROM `creature_equip_template` WHERE `entry`=123;
+INSERT INTO `creature_equip_template` (`entry`,`equipentry1`,`equipentry2`,`equipentry3`) VALUES
+(123,0,0,0);
+UPDATE `creature` SET `equipment_id`=123 WHERE `guid` IN(57895,57889,58153,57942,58142,58149,58154,57904); -- Use an empty equip_template
+
+-- ----------------------------------------------------------
+-- Father Malgor Devidicus - Drink emote
+-- ----------------------------------------------------------
+UPDATE `creature_template` SET `AIName`='EventAI' WHERE `entry`=16825;
+
+-- ----------------------------------------------------------
+-- Honor Hold Defenders
+-- ----------------------------------------------------------
+
+-- Outhouse guard GUID: 57942
+-- Since this guard has a gossip menu it needs to have a separate template from the other guards (as far as I know). 
+-- So what I did was I gave this one the template of the two talking guards outside the destroyed keep, and I gave them the normal guard template.
+-- I see no reason why those two would need a separate template anyways, so I thought it was better to reuse this one than create a separate one for the outhouse guard.
+
+UPDATE `creature` SET `id`=16842 WHERE `guid` IN(72636,72637);
+UPDATE `creature` SET `id`=20513,`spawndist`=0,`MovementType`=2 WHERE `guid`=57942;
+UPDATE `creature_template` SET `GossipMenuId`=20026,`NpcFlags`=1 WHERE `entry`=20513;

```

Comment:
That might just be the ID we used on Hellfire and I missed to change it here. My bad